Makefile

```diff
@@ -432,15 +432,20 @@ npm-update: npm-check
 	rm -rf node_modules package-lock.json
 	npm install --package-lock
 
+.PHONY: webpack
+webpack: npm
+	npm install --save-dev webpack webpack-cli
```

Comment:
This is unnecessary. Webpack will be installed when `npm install` is run.

Reply:
done.